src/Equinox.MessageDb/Tracing.fs

```diff
@@ -4,33 +4,20 @@ module internal Equinox.MessageDb.Tracing
 open Equinox.MessageDb.Core
 open System.Diagnostics
 
-[<AbstractClass; Sealed; System.Runtime.CompilerServices.Extension>]
-type ActivityExtensions private () =
+type private Activity with
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddExpectedVersion(act: Activity, version) =
-        match version with StreamVersion v -> act.AddTag("eqx.expected_version", v) | Any -> act
+    member act.AddExpectedVersion(version) =
+        match version with StreamVersion v -> act.SetTag("eqx.expected_version", v) | Any -> act
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddLastVersion(act: Activity, version: int64) =
-        act.AddTag("eqx.last_version", version)
+    member act.AddLastVersion(version: int64) = act.SetTag("eqx.last_version", version)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddBatchSize(act: Activity, size: int64) =
-        act.AddTag("eqx.batch_size", size)
+    member act.AddBatchSize(size: int64) = act.SetTag("eqx.batch_size", size)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddBatches(act: Activity, batches: int) =
-        act.AddTag("eqx.batches", batches)
+    member act.AddBatches(batches: int) = act.SetTag("eqx.batches", batches)
 
-    [<System.Runtime.CompilerServices.Extension>]
-    static member AddStartPosition(act: Activity, pos: int64) =
-        act.AddTag("eqx.start_position", pos)
+    member act.AddStartPosition(pos: int64) = act.SetTag("eqx.start_position", pos)
```

Comment:
position doesnt talk to me - is it a standard term? index? version?
Oh wait this is -1 based nonsense is it :P

Reply:
one liner comments would help a lot (also for `version`, is it v0 if 0 events (please say yes, I'd like to standardise it across stores)?

Reply:
I'll get a documentation table going for tags and their meaning 👍

Reply:
Your scheme seems to be working well (but having a literals module with one liner xmldoc should replace it pretty soon; resolving

Reply:
aligned everything with version terminology